### DISCUSSION TITLE
使用 Obsidian - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#28] [Announcements]
# 使用 Obsidian - forecho's Blog

引言
Obsidian 是一个非常好用的笔记软件，使用它有一阵子了，今天打算给大家分享一下我的使用经验。
为什么使用 Obsidian
因为要记笔记，工作中我们常常会遇到一些问题，而且有些问题我们不止一次遇到，我有记录问题如何解决的习惯，下次一旦遇到重复的问题，我就能快速解决了。
生活中也会常常记录自己的思考。把这些思考的内容记录下来，日积月累，就能形成自己的知识体系。
书写是为了更好地思考。
为什么选择 Obsidian
我之前使用过很多笔记软件，都不太满意，之前最常用的是印象笔记了，非常喜欢它的浏览器剪辑功能，但是最终还是被它恶心到了，烦人的广告（开了会员也没用），臃肿的客户端。

https://blog.forecho.com/use-obsidian.html

**Comment by Cathay-Chen** [comment]
推荐阅读: https://sspai.com/post/72385
